wcfsetup/install/files/lib/system/user/notification/event/ModerationQueueCommentResponseUserNotificationEvent.class.php

```diff
@@ -48,9 +48,10 @@ class ModerationQueueCommentResponseUserNotificationEvent extends AbstractShared
     protected $moderationQueueLoaded = false;
 
     /**
-     * @inheritDoc
+     * language item for the type name
+     * @var string|null
      */
-    protected $stackable = true;
+    protected $typeName;
```

Comment:
This should be typed as `protected string $typeName;` (with the `@var` removed).